hack/lib/tracing.bash

```diff
@@ -10,8 +10,6 @@ function install_tracing {
     install_zipkin_tracing
   elif [[ "${TRACING_BACKEND}" == "tempo" ]]; then
     install_tempo_tracing
-  elif [[ "${TRACING_BACKEND}" == "otel" ]]; then
-    install_opentelemetry_tracing
```

Comment:
isn't this also used for regular non SM tests ? /cc @mgencur @maschmid

Reply:
We should be using tempo instead of otel now everywhere. (Jaeger is deprecated)

Reply:
Yepp, we never set `otel` anywhere anymore. Tempo is used as default.